CMakeLists.txt

```diff
@@ -16,72 +22,56 @@ FOREACH (p
     ENDIF ()
 endforeach ()
 
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake) # project specific cmake dir
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/../cmake)
-
-project(Monitoring)
-
-# Load some basic macros which are needed later on
-include(O2Utils)
-include(MonitoringDependencies)
-
-# Set the default build type to "RelWithDebInfo"
-if(NOT CMAKE_BUILD_TYPE)
-    set(CMAKE_BUILD_TYPE "RelWithDebInfo"
-            CACHE
-            STRING "Choose the type of build, options are: Debug Release RelWithDebInfo MinSizeRel Coverage."
-            FORCE)
-endif(NOT CMAKE_BUILD_TYPE)
```

Comment:
I would keep the default build type to `RelWithDebInfo` if not defined to avoid having an undefined one.